### ISSUE TITLE
invalid serialized PackageId for key `package.dependencies`

### ISSUE THREAD

**Issue by brenzi** [sfackler/cargo-tree#80]
`cargo tree` fails with
```
error: failed to parse lock file at: /home/abrenzikofer/test_no_std/Cargo.lock

Caused by:
  invalid serialized PackageId for key `package.dependencies`

```
for the following crate:
https://github.com/scs/test-no-std/commit/e584d5711f1d0d55e9fde21bf0c1edad463ea856

**Comment by Rua** [comment]
This happened to me as well.

**Comment by jplatte** [comment]
Just a guess, but this might have to do with https://github.com/rust-lang/cargo/pull/7579. Does deleting the lock file and re-generating it with an older cargo fix the issue?

**Comment by est31** [comment]
#85 will fix this. You can use @lzutao 's fork for the meantime:

```
cargo install --force --git https://github.com/lzutao/cargo-tree --branch bump-deps
```

**Comment by est31** [comment]
Fixed as of release 0.28.0. ping @sfackler issue can be closed.
